cmd/wait/error.go

```diff
@@ -27,5 +27,8 @@ func IsInvalidFlag(err error) bool {
 var serverErrorPattern = regexp.MustCompile(`an error on the server .*`)
 
 func IsServerError(err error) bool {
+	if err == nil {
```

Comment:
This causes a panic if we don't check for `nil` before calling the `.Error()` method.